org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/ChangeConverter2.java

```diff
@@ -120,7 +87,8 @@ protected void _handleReplacements(IEmfResourceChange change) {
 				Document document = context.getDocument();
 				Range range = new Range(document.getPosition(0), document.getPosition(document.getContents().length()));
 				TextEdit textEdit = new TextEdit(range, newContent);
-				return addTextEdit(uri, document, textEdit);
+				addTextEdit(uri, document, textEdit);
+				return null;
 			}).get();
 		} catch (InterruptedException | ExecutionException | IOException e) {
 			throw Exceptions.sneakyThrow(e);
```

Comment:
Not your code, but `sneakyThrow` is evil. It will disguise checked Exceptions as unchecked exceptions and throw them without mentioning the checked exceptions in a function's signature.

Since you have to edit the signature to correctly handle this, not sure if there's a good solution here

Reply:
Thanks. I would be happy changing it, but since it is unrelated to my changes, I would leave it be unless someone wants me to do the change.